### PR TITLE
Update README.md to specify gathered telemetry

### PR DESCRIPTION
**Pull request by JamesWTruher**
# PR Summary

The telemetry statement in the README seems out of date.

## PR Context

The telemetry statement should fully describe the data we gather.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
 
 ### Telemetry
 
-By default, PowerShell collects the OS description and the version of PowerShell (equivalent to `$PSVersionTable.OS` and `$PSVersionTable.GitCommitId`) using [Application Insights](https://azure.microsoft.com/services/application-insights/).
-To opt-out of sending telemetry, create an environment variable called `POWERSHELL_TELEMETRY_OPTOUT` set to a value of `1` before starting PowerShell from the installed location.
+Sending telemetry is controlled via an environment variable called `POWERSHELL_TELEMETRY_OPTOUT`.
+Set the value to `1`, `yes`, or `true` before starting PowerShell to disable telemetry.
+Setting the value to `0`, `no`, `false`, or not creating the variable will enable telemetry.
```
Comment (SteveL-MSFT) [inline README.md:245]: Isn't it more accurate that if it's anything other than `1`,`yes`, or `true` then it defaults to false meaning to send telemetry?

File: README.md [hunk 2]
```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
...
 The telemetry we collect falls under the [Microsoft Privacy Statement](https://privacy.microsoft.com/privacystatement/).
 
+The following is sent when the shell starts:
+
+- Operating system information
```
Comment (SteveL-MSFT) [inline README.md:250]: it would be great to make these links to the source code where we collect this information

File: README.md [hunk 3]
```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
...
+- The commit id of the build
+- The enabled experimental features
+- A randomly generated guid representing the session
+- A randomly generated guid representing the system
+- Geo-location information is supplied by Application Insights (Country, Region, State, City, if available)
+- The value of the environment variable `POWERSHELL_DISTRIBUTION_CHANNEL`
+
+During shell operation the following is sent periodically:
+
+- The name of a module when loaded if it is in the list of modules (see line 128 in src/System.Management.Automation/utils/Telemetry.cs).
```
Comment (SteveL-MSFT) [inline README.md:260]: make that a link instead of text to the source line

Comment (SteveL-MSFT) [inline README.md:260]:
```suggestion
- The name of a module when loaded if it is in the pre-approved list of modules (see line 128 in src/System.Management.Automation/utils/Telemetry.cs).
```

File: README.md [hunk 4]
```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
...
+Modules which are not in this predetermined list are reported as "Anonymous".
+- The type of item being executed (Alias, Function, Filter, Cmdlet, ExternalScript, Application, Script, or Configuration).
+We do not collect the name of the executed item, the parameters used, or values for any parameter.
+- When a `PowerShell` object is created
+- When a remote session is created
+
+In addition to the caveats above, the following is _not_ collected:
```
Comment (SteveL-MSFT) [inline README.md:267]:
```suggestion
In addition to the caveats above, no personally identifiable information is collected.  For example, the following is _not_ collected:
```

File: README.md (hunk 3 above)
Comment (daxian-dbw) [reply README.md]: Make sure to use the [permanent links to files](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files).

File: README.md (hunk 2 above)
Comment (daxian-dbw) [reply README.md:250]: Make sure to use the [permanent links to files](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files).

File: README.md [hunk 5]
```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
...
+- The commit id of the build
+- The enabled experimental features
+- A randomly generated guid representing the session
+- A randomly generated guid representing the system
+- Geo-location information is supplied by Application Insights (Country, Region, State, City, if available)
+- The value of the environment variable `POWERSHELL_DISTRIBUTION_CHANNEL`
+
+During shell operation the following is sent periodically:
+
+- The name of a module when loaded if it is in the pre-approved list of modules (see line 128 in src/System.Management.Automation/utils/Telemetry.cs).
```
Comment (StevenBucher98) [inline README.md:260]:
```suggestion
- The name of a module when loaded if it is in the pre-approved [list of modules](https://github.com/PowerShell/PowerShell/blob/d74c6bfb7bc2ff84ae15cceeba2fb7dc5c3ccd17/src/System.Management.Automation/utils/Telemetry.cs#L128) 
```

Comment (StevenBucher98) [reply README.md:260]: If it is not best practice to include this line number perma-link then feel free to ignore.

File: README.md [hunk 6]
```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
...
+Modules which are not in this predetermined list are reported as "Anonymous".
+- The type of item being executed (Alias, Function, Filter, Cmdlet, ExternalScript, Application, Script, or Configuration).
```
Comment (daxian-dbw) [inline README.md:262]: Just out of curiosity, how useful do we think it is to collect the command type information?

File: README.md (hunk 2 above)
Comment (JamesWTruher) [reply README.md:250]: @SteveL-MSFT - this could be problematic if the code changes

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by daxian-dbw** [comment]
Pointing to the topic doc is surely better.

File: README.md (hunk 6 above)
Comment (daxian-dbw) [reply README.md]: @JamesWTruher can you please comment on this question? I'm just curious how useful this information turns out to be in our telemetry analysis so far.

**Review by sdwheeler — APPROVED** [review]

**Review by StevenBucher98 — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@JamesWTruher please fix up the spelling issue, which caused the 'static-analysis' CI failed.

**Review by daxian-dbw — CHANGES_REQUESTED** [review]
'static-analysis' CI failed, need to fix the spelling issue.

**Comment by TravisEz13** [comment]
I updated the CI yamls so future PRs to readme and spelling shouldn't trigger product tests, but because the PR now touches those files, we need to run those pipelines to validate those changes.

**Review by sdwheeler — APPROVED** [review]
Changes look good to me.

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**